pkg/workflows/sdk/runtime.go

```diff
@@ -2,27 +2,24 @@ package sdk
 
 import (
 	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
-	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 )
 
 // BreakErr can be used inside the compute capability function to stop the execution of the workflow.
 var BreakErr = capabilities.ErrStopExecution
 
 type MessageEmitter interface {
 	// Emit sends a message to the labeler's destination.
-	Emit(string) error
+	Emit(string)
```

Comment:
is the `ModuleConfig.MessageEmitter` also changing? This is the interface that the host (e.g., core node) implements and provides to the wasm runtime.